projects/igniteui-angular/src/lib/grids/pivot-grid/public_api.ts

```diff
@@ -12,6 +12,70 @@ export * from './pivot-grid.directives';
 export * from '../../data-operations/pivot-strategy';
 export * from '../../data-operations/pivot-sort-strategy';
 
+/* Imports that cannot be resolved from IGX_GRID_COMMON_DIRECTIVES spread
+    NOTE: Do not remove! Issue: https://github.com/IgniteUI/igniteui-angular/issues/13310
+*/
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {
+    IgxRowDirective,
+    IgxGridFooterComponent,
+    IgxAdvancedFilteringDialogComponent,
+    IgxHeaderCollapsedIndicatorDirective,
+    IgxHeaderExpandedIndicatorDirective,
+    IgxRowCollapsedIndicatorDirective,
+    IgxRowExpandedIndicatorDirective,
+    IgxSortAscendingHeaderIconDirective,
+    IgxSortDescendingHeaderIconDirective,
+    IgxSortHeaderIconDirective,
+    IgxExcelStyleHeaderIconDirective,
+    IgxDragIndicatorIconDirective,
+    IgxRowDragGhostDirective,
+    IgxGridStateDirective,
+    IgxGridHeaderComponent,
+    IgxGridHeaderGroupComponent,
+    IgxGridHeaderRowComponent,
+    IgxFilterCellTemplateDirective,
+    IgxSummaryTemplateDirective,
+    IgxCellTemplateDirective,
+    IgxCellValidationErrorDirective,
+    IgxCellHeaderTemplateDirective,
+    IgxCellFooterTemplateDirective,
+    IgxCellEditorTemplateDirective,
+    IgxCollapsibleIndicatorTemplateDirective,
+    IgxColumnComponent,
+    IgxColumnGroupComponent,
+    IgxColumnLayoutComponent,
+    IgxColumnActionsComponent,
+    IgxColumnHidingDirective,
+    IgxColumnPinningDirective,
+    IgxRowSelectorDirective,
+    IgxGroupByRowSelectorDirective,
+    IgxHeadSelectorDirective,
+    IgxCSVTextDirective,
+    IgxExcelTextDirective,
+    IgxGridToolbarActionsComponent,
+    IgxGridToolbarAdvancedFilteringComponent,
+    IgxGridToolbarComponent,
+    IgxGridToolbarExporterComponent,
+    IgxGridToolbarHidingComponent,
+    IgxGridToolbarPinningComponent,
+    IgxGridToolbarTitleComponent,
+    IgxGridToolbarDirective,
+    IgxGridExcelStyleFilteringComponent,
+    IgxExcelStyleHeaderComponent,
+    IgxExcelStyleSortingComponent,
+    IgxExcelStylePinningComponent,
+    IgxExcelStyleHidingComponent,
+    IgxExcelStyleSelectingComponent,
+    IgxExcelStyleClearFiltersComponent,
+    IgxExcelStyleConditionalFilterComponent,
+    IgxExcelStyleMovingComponent,
+    IgxExcelStyleSearchComponent,
+    IgxExcelStyleColumnOperationsTemplateDirective,
+    IgxExcelStyleFilterOperationsTemplateDirective,
+    IgxExcelStyleLoadingValuesTemplateDirective
+} from "../public_api";
```

Comment:
I believe quite a few of these relate to features that the pivot doesn't really support, so the common directives are not _that_ common and could use a further breakdown, but since this is just to properly resolve what's already there it should be fine.